src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -4391,6 +4391,13 @@ interface UserInterface {
    * RAM cost: 0 GB
    */
   getGameInfo(): GameInfo;
+
+  /**
+   * Clear the Terminal window, as if the player ran `clear` in the terminal
+   * @remarks
+   * RAM cost: 0 GB
```

Comment:
Should this be `* RAM cost: 0.2 GB`? The constant RAM cost is set to `0.2`, which I believe should match

Reply:
Changed. Was apparently missed due to me going back and forth on RAM usage a few times. Woops!